src/Gir.Tests/RecordTests.cs

```diff
@@ -110,7 +131,10 @@ public struct ByteArray
 		/// The following bytes are moved down one place.
 		///</summary>
 		///<returns>the #GByteArray</returns>
-		public ByteArray RemoveIndex (uint index_);
+		public ByteArray RemoveIndex (uint index_)
```

Comment:
This was here before, but we probably should make a note to fix this

Reply:
The underscore?

Reply:
sorry, yes. We probably shouldn't have variable names `index_` :)